src/main/resources/db/migration/v0.36.0.4__highthroughputexpressiondataset.sql

```diff
@@ -0,0 +1,103 @@
+-- To create bulk loads
+
+INSERT INTO bulkloadgroup (id, name) VALUES (nextval('bulkloadgroup_seq'), 'HTP Expression Dataset Annotation Bulk Loads');
+
+INSERT INTO bulkload (id, backendbulkloadtype, name, bulkloadstatus, group_id)
+SELECT nextval('bulkload_seq'), 'HTP_EXPRESSION_DATASET_ANNOTATION', 'FB HTP Expression Dataset Annotation Load', 'STOPPED', id FROM bulkloadgroup WHERE name = 'HTP Expression Dataset Annotation Bulk Loads';
+INSERT INTO bulkload (id, backendbulkloadtype, name, bulkloadstatus, group_id)
+SELECT nextval('bulkload_seq'), 'HTP_EXPRESSION_DATASET_ANNOTATION', 'ZFIN HTP Expression Dataset Annotation Load', 'STOPPED', id FROM bulkloadgroup WHERE name = 'HTP Expression Dataset Annotation Bulk Loads';
+INSERT INTO bulkload (id, backendbulkloadtype, name, bulkloadstatus, group_id)
+SELECT nextval('bulkload_seq'), 'HTP_EXPRESSION_DATASET_ANNOTATION', 'MGI HTP Expression Dataset Annotation Load', 'STOPPED', id FROM bulkloadgroup WHERE name = 'HTP Expression Dataset Annotation Bulk Loads';
+INSERT INTO bulkload (id, backendbulkloadtype, name, bulkloadstatus, group_id)
+SELECT nextval('bulkload_seq'), 'HTP_EXPRESSION_DATASET_ANNOTATION', 'RGD HTP Expression Dataset Annotation Load', 'STOPPED', id FROM bulkloadgroup WHERE name = 'HTP Expression Dataset Annotation Bulk Loads';
+INSERT INTO bulkload (id, backendbulkloadtype, name, bulkloadstatus, group_id)
+SELECT nextval('bulkload_seq'), 'HTP_EXPRESSION_DATASET_ANNOTATION', 'SGD HTP Expression Dataset Annotation Load', 'STOPPED', id FROM bulkloadgroup WHERE name = 'HTP Expression Dataset Annotation Bulk Loads';
+INSERT INTO bulkload (id, backendbulkloadtype, name, bulkloadstatus, group_id)
+SELECT nextval('bulkload_seq'), 'HTP_EXPRESSION_DATASET_ANNOTATION', 'WB HTP Expression Dataset Annotation Load', 'STOPPED', id FROM bulkloadgroup WHERE name = 'HTP Expression Dataset Annotation Bulk Loads';
+
+INSERT INTO bulkscheduledload (id, cronschedule, scheduleactive)
+SELECT id, '0 0 22 ? * SUN-THU', false FROM bulkload WHERE backendbulkloadtype = 'HTP_EXPRESSION_DATASET_ANNOTATION';
```

Comment:
Can we set the scheduleactive field to be true for these - otherwise we have to manually set them to true on each environment after deployment.  Previous migrations had false as at the time we had turned scheduling off.

Reply:
Sure